docs/lib/storage/fragments/native_common/getting-started/common.md

```diff
@@ -8,7 +8,7 @@ To setup and configure your application with Amplify Storage and go through a si
 <inline-fragment platform="ios" src="~/lib/storage/fragments/ios/getting-started/10_preReq.md"></inline-fragment>
 <inline-fragment platform="android" src="~/lib/storage/fragments/android/getting-started/10_preReq.md"></inline-fragment>
 
-## Provision Backend Storage Services
+## Provision backend storage
```

Comment:
This is a 2nd level heading -- shouldn't it get `Title Case`?

Reply:
Whoa. I thought we weren't doing title case on second level headings. Do we have these styles defined somewhere?

Reply:
You're right, sorry: https://cirosantilli.com/markdown-style-guide#header-case